### ISSUE TITLE
Save annotation name changes to both gene and mRNA features

### ISSUE THREAD

**Issue by MoffMade** [TAMU-CPT/Apollo#16]
For example:
![image](https://user-images.githubusercontent.com/5633348/94932555-da580200-048e-11eb-88cd-329c41b1065e.png)

Some confusion is caused by updates to the mRNA name are displayed in Apollo but later processing steps use the gene feature name. 

Possible fixes:
- Copy changes to gene name to mRNA name when changed(we used to do this in our old version via hacky steps on the annotation editor popup)
- Configurable in Apollo to display/edit gene feature names instead of displaying mRNA names
- Post-retrieval tool configurable to rename mRNA/gene to match
- Change the retrieval process to rename mRNA and gene to be matching (unideal)

**Comment by jrr-cpt** [comment]
Submitted a question for Nathan in Apollo [repo](https://github.com/GMOD/Apollo/issues/2519)

**Comment by MoffMade** [comment]
Discussed in meeting, track further work in linked issue above

**Comment by MoffMade** [comment]
Nathan finished up the sync button code in the main trunk, i've merged it and our current 2.6.1.cpt branch as '2.6.1.cpt-1' so that we can test it before roll out. 

Docker image is building on quay.io right now, and will pull it to apollo-testing as soon as it's done

**Comment by MoffMade** [comment]
Sync Name button available in https://cpt.tamu.edu/apollo-testing

I found a couple of oddities so far that I'll report to Nathan since I think the use can be better:
- The "Sync Name to gene/transcript" button does not become active/clickable until you change the gene or mRNA name value.
- The updated value on the target feature (the one you are copying/syncing the name TO) does not appear until the page is refreshed which is unideal
- The button is not active when the names are different but have not been immediately recently changed

![image](https://user-images.githubusercontent.com/5633348/95616171-ff201c80-0a2e-11eb-8ceb-5aa3749445a2.png)

**Comment by MoffMade** [comment]
ISSUES:

- Accessing the Annotation Editor for mRNA annotations 
Sometimes when clicking on "Open Annotation" on a feature through Right Click menu, sometimes the editor window does not load directly unless you open the gene first and THEN the mRNA, console shows a js error. Transient and difficult to reproduce, seems to happen mostly when recently logged in. Several members have reported this from both Chrome and Firefox, and varying levels of frequency. 

- Annotation window closes with a js error in console after syncing name from either gene or transcript to the other (although the change does seem to sync correctly if reloaded), again seemingly transient. This doesn't crash every time. Tested on Chrome
```
Uncaught zt {e: Rh, f: "Exception caught: (TypeError) : Cannot read property 'C' of null", g: null, __gwt$backingJsError: Error: Exception caught: (TypeError) : Cannot read property 'C' of null
    at gj.dj [as Ib] (annot…, a: kDb}
```

- Sometimes, when changing name in gene or transcript or also syncing name to other feature, the list will refresh but changes will not be displayed unless list is refreshed again (either full page refresh or changing tab to tracks and back for example)

- mRNA name changes will be immediately displayed on the Annotation Track in jBrowse but not updated in the Annotations list in the Apollo UI

- gene or mRNA change can be input and accepted, but will not refresh in the UI, allowing for users to try again but the UI will not accept the change again until reloaded.

**Comment by MoffMade** [comment]
https://github.com/GMOD/Apollo/issues/2531

**Comment by MoffMade** [comment]
Changed merged into branch 2.6.1.cpt-3 and pushed to live
